modules/ocf/manifests/serial_getty.pp

```diff
@@ -0,0 +1,8 @@
+class ocf::serial_getty {
+  service { 'serial-getty@ttyS0':
+    ensure   => running,
```

Comment:
@daradib that's what i tried, but it didn't actually start it :\

```puppet
exec { 'systemctl enable --now serial-getty@ttyS0':
  creates => '/etc/systemd/system/getty.target.wants/serial-getty@ttyS0.service',
} ~>
service { 'serial-getty@ttyS0': provider => systemd }
```

does enable it, but just leaves it stopped (though it does notify it, not sure exactly what it's doing)